### ISSUE TITLE
PHP 8.3

### ISSUE THREAD

**Issue by luigifab** [adminerevo/adminerevo#60]
Nice to see an update for Adminer.

There are a lot of `Warning: Trying to access array offset on null in ...` with PHP 8.3.
Can you... check them?

**Comment by LionelLaffineur** [comment]
Dear @luigifab, thanks for sharing this.
Since I'm using adminer daily, it seems that I'm not working in the same environment as you do.
I'm mainly using PHP 8.2.7 and MySQL (MariaDB).  Would you give me more details about your environment ?
You can also be very helpful by using the latest version of the main branch and report all the warnings you encounter with the complete stack trace and URL.
Regards,
L.

**Comment by luigifab** [comment]
Yes, PHP 8.3.0-rc3 with MariaDB... but by checking the main branch, I see that there are no warnings.
So I checked, this is because to run AdminerEvo I use:

```php
error_reporting(E_ALL);
ini_set('display_errors', 1);
function adminer_object() {
	require_once('plugin.php');
	require_once('shortcuts.php');
	require_once('colorfields.php');
	$plugins = [
		new Shortcuts(),
		new AdminerColorfields()
	];
	return new AdminerPlugin($plugins);
}
require_once('adminer.php');
```

And result is:
![image](https://github.com/adminerevo/adminerevo/assets/31816829/8aead2ae-8057-4763-93e9-b265bed64afe)

---


With the main branch, I have warnings by adding the two first lines in _adminer/index.php_, but not for the login page:

![image](https://github.com/adminerevo/adminerevo/assets/31816829/31571a6c-fe54-411a-90ed-10d30ce535c1)

**Comment by LionelLaffineur** [comment]
Hi @luigifab, I don't like the idea of hiding warnings, even if I understand the idea of a quick and easy fix.
I prefer to fix them all, one by one, to make the code better.  I've already started.
Any report of that kind of warning is welcome !
Regards,
L.
